src/main/resources/js/moduleProvidedLibs.js

```diff
@@ -0,0 +1,132 @@
+let global = this;
+
+// Extra libs
+global.ArrayList = Java.type("java.util.ArrayList");
+global.HashMap = Java.type("java.util.HashMap");
+global.Keyboard = Java.type("org.lwjgl.input.Keyboard");
+global.ReflectionHelper = Java.type("net.minecraftforge.fml.relauncher.ReflectionHelper");
+
+// Triggers
+global.TriggerRegister = Java.type("com.chattriggers.ctjs.engine.langs.js.JSRegister").INSTANCE;
+global.TriggerResult = Java.type("com.chattriggers.ctjs.triggers.OnTrigger.TriggerResult");
+global.Priority = Java.type("com.chattriggers.ctjs.triggers.OnTrigger.Priority");
+//#if MC<=10809
+global.InteractAction = Java.type("net.minecraftforge.event.entity.player.PlayerInteractEvent.Action");
+//#else
+//$$ global.InteractAction = Java.type("com.chattriggers.ctjs.minecraft.listeners.ClientListener").INSTANCE.PlayerInteractAction;
+//#endif
+
+// Libraries
+global.ChatLib = Java.type("com.chattriggers.ctjs.minecraft.libs.ChatLib");
+global.EventLib = Java.type("com.chattriggers.ctjs.minecraft.libs.EventLib");
+
+global.Renderer = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Renderer");
+global.Shape = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Shape");
+global.Rectangle = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Rectangle");
+global.Text = Java.type("com.chattriggers.ctjs.minecraft.libs.renderer.Text");

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Java' is not defined.](https://app.codacy.com/manual/FalseHonesty/ct.js/pullRequest?prid=5138651)